release/karaf/itests/pom.xml

```diff
@@ -28,6 +28,7 @@
     <description>Fuse Integration Release Karaf Tests</description>
 
     <modules>
+       <module>camel-module</module>
```

Comment:
Why all of these tests are included in a new subproject camel-module? There is already another module called camel. In my opinion should be placed there.